controller/index.js

```diff
@@ -1,9 +1,14 @@
 const model = require('../model/index.js');
+const utils = require('../utils/index.js');
 
 let seller = {
   create: model.post,
   read: model.get,
-  update: model.put,
+  update: (args, updateInfo) => {
```

Comment:
why build the query within this file when your other methods reference the queries in the model file? (this might just be a misunderstanding on my part w/ the model > controller flow)